packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -16,10 +16,39 @@ class _BaseConfig:
     keep_untouched = (cached_property,)
 
 
+class ContainerSpec(BaseModel):
+    """Implements entries that can be overriden for https://docs.docker.com/engine/api/v1.41/#operation/ServiceCreate
+    request body: TaskTemplate -> ContainerSpec
+    """
+
+    command: list[str] = Field(
+        alias="Command",
+        description="Used to override the container's command",
+        # NOTE: currently constraint to our use cases. Might mitigate some security issues.
+        min_items=1,
+        max_items=2,
```

Comment:
Even with the blow examples It's tricky to understand why only max 2 arguments. Usually users have more complex cli.

Reply:
As the note suggests ... I will constraint for the moment to our use case